spec/models/physical_server_profile_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe PhysicalServerProfile do
+  subject { FactoryBot.create(:physical_server_profile) }
+
+  describe "#queue_name_for_ems_operations" do
+    context "with an active configured_system" do
+      let(:manager)         { FactoryBot.create(:physical_infra) }
```

Comment:
If we're going to pad with spaces might as well align with the block below

Reply:
Fixed in the latest commit